### ISSUE TITLE
Auto embed tweets in The Latest

### ISSUE THREAD

**Issue by jonahpriorities** [jakewelchert/Priorities-USA---static#9]
Is it possible to auto post our tweets as posts in The Latest. Because we don't think we will have frequent content there was a request, this may have also been a request before my time, to have tweets auto post. I mentioned having a latest tweet as a separate module below The Latest but we think it would be better in The Latest.

**Comment by jonahpriorities** [comment]
If the above is doable, is it possible to embed images/video that are included in the tweets?

**Comment by jakewelchert** [comment]
Are you describing the the latest feed on the homepage, or tweets populating on that actual "The Latest' archive page?

If it's the archive page, I would suggest not placing tweets there and just focus on posts that Priorities provide. We want to push people back to twtitter, so seeing tweets on the homepage will encourage that.

**Comment by jonahpriorities** [comment]
Latest feed on the homepage.

**Comment by jakewelchert** [comment]
Yes, we should be able to auto-populate tweets. On the current version of the homepage, the tweet module is dark blue. We can include as many as we need into that feed.

Also, with images and video, we always had just the tweet copy in mind since 80% of the time that's all that's included. How can we make this work? If every time you tweet a video, will you want to post about it on the site?

**Comment by jonahpriorities** [comment]
I talked to Tara about this possible issue yesterday. We will be building visual content just for twitter so ideally we would like it to display but it is likely a nice to have rather than a requirement at this point. I don't want this holding up other dev but I think Priorities would want to revisit it at some point so we look to you for guidance on how complicated it would be.

**Comment by jakewelchert** [comment]
Sounds great. I'll think about this some more and see what we can come up with.

**Comment by jakewelchert** [comment]
Here's a screenshot of how those photos and videos could work, but we're going to have to make them background images so they're full height. This will make for a funny crop, but I think that's okay if we have some sort of overlay.

![screen shot 2015-10-30 at 2 31 51 pm](https://cloud.githubusercontent.com/assets/3301072/10856132/8374d52c-7f13-11e5-990c-8230ffdaf174.png)

**Comment by jonahpriorities** [comment]
I'm not thrilled with that solution or cropping. If a crop makes one of the graphics look strange it opens the potential of bad media attention for us. That option is probably out. Can we treat it how we treat a post with image, so the image is half the space and auto center it vertically. See below:

![image](https://cloud.githubusercontent.com/assets/15385469/10896129/09219506-8186-11e5-8eb3-53b6a90cec3e.png)
